ios/Classes/PluginExtensions/SwiftOneginiPlugin+Auth.swift

```diff
@@ -44,61 +45,73 @@ extension SwiftOneginiPlugin: OneginiPluginAuthProtocol {
             OneginiModuleSwift.sharedInstance.authenticateUser(nil, callback: result)
         }
     }
-    
+
+    func authenticateUserImplicitly(_ call: FlutterMethodCall, _ result: @escaping FlutterResult) {
+        guard let _arg = call.arguments as! [String: Any]? else { return }
```

Comment:
1. just `arg` or `arguments` 
2. `as?` instead of `as!`
3. no result with some error needed in this case?